api/src/Entity/User.php

```diff
@@ -74,7 +74,7 @@ class User implements UserInterface
     /**
      * @var int
      * @JMS\Type("integer")
-     * @JMS\Groups({"user", "report-submitted-by", "user-id", "user-list"})
+     * @JMS\Groups({"user", "report-submitted-by", "user-id", "user-list", "admin-account-reports"})
```

Comment:
Seeing as all of the properties have `user` and `user-list` as JMS groups that you need you could try using this when calling out the the API rather than adding your own as the group lists can start to bloat quite quickly when adding them for individual functions. It's worth checking if this impacts performance with prod-like data.